<launch>
  <node pkg="rospy_crazyflie" name="crazyflie_server" type="server" output="screen">
    <rosparam file='$(find rospy_crazyflie)/config/config.yaml' command='load'/>
  </node>
</launch>
